src/libraries/Common/src/System/Net/Security/NegotiateStreamPal.Unix.cs

```diff
@@ -299,9 +299,9 @@ Interop.NetSecurityNative.Status status
         internal static SecurityStatusPal AcceptSecurityContext(
             SafeFreeCredentials? credentialsHandle,
             ref SafeDeleteContext? securityContext,
-            ContextFlagsPal requestedContextFlags,
+            ContextFlagsPal _ /*requestedContextFlags*/,
             ReadOnlySpan<byte> incomingBlob,
-            ChannelBinding? channelBinding,
+            ChannelBinding? _1 /*channelBinding*/,
```

Comment:
Nit: This looks ugly to me.  I'd rather we just suppress here.

Reply:
Fixed